sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -72,6 +72,20 @@
     private static final int DEFAULT_QUERY_PLAN_RESPONSE_TIMEOUT_IN_SECONDS = 5;
     private static final int DEFAULT_ADDRESS_REFRESH_RESPONSE_TIMEOUT_IN_SECONDS = 5;
 
+    // SessionTokenMismatchRetryPolicy Constants
+    // TODO fabianm - do we really want to keep these config overrides like in .Net
```

Comment:
In .Net we allow overriding these defaults via Environment variables - this is the reason why I added it here as well. But to be honest I don't expect this to be an often used extensibility - most likely the configuration was allowed in .Net for Compute/Mongo/Cassandra - so I would be ok with removing and hard-coding the settings instead as well. Let me know what you think...